src/adb.js

```diff
@@ -32,11 +32,23 @@ const DEFAULT_EXEC = (args, callback) => {
     callback
   );
 };
+
+// AW : Allow execution of custom command, not only adb
```

Comment:
![](https://imgs.xkcd.com/comics/turbine.png)

Maybe we can find a different way, this will not work on all platforms.

Reply:
Function removed as not needed anymore. done